packages/editor/src/client/panel/publishArticlePanel.tsx

```diff
@@ -73,15 +77,15 @@ export function PublishArticlePanel({
         {pendingPublishDate && (
           <Box marginBottom={Spacing.Small}>
             <Typography variant="subtitle1" color="alert">
-              There is already a pending publication scheduled at{' '}
+              {t('There is already a pending publication scheduled at')}{' '}
```

Comment:
You should use formatting for this whole Typography block => https://www.i18next.com/translation-function/formatting

Reply:
Done